authority.gemspec

```diff
@@ -22,7 +22,7 @@ Gem::Specification.new do |spec|
   spec.files         = `git ls-files`.split("\n")
   spec.files -= spec.files.select { |f| f.match(%r{^(bin|spec|features|gemfiles)/}) }
   spec.test_files    = `git ls-files -- {test,spec,features}/*`.split("\n")
-  spec.name          = 'civis-authority'
+  spec.name          = 'authority'
```

Comment:
yup. the only 2 changes are the renaming of this file from 
civis-authority.gemspec => authority.gemspec

and this 1 line of code.